cyber/time/time.cc

```diff
@@ -83,6 +83,10 @@ bool Time::IsZero() const { return nanoseconds_ == 0; }
 
 uint64_t Time::ToNanosecond() const { return nanoseconds_; }
 
+uint64_t Time::ToMicrosecond() const {
```

Comment:
Maybe a `double Time::ToDoubleMicrosecond() const` will be needed in the future.